ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/fileinput/v2/fileinput/README.md

```diff
@@ -0,0 +1,93 @@
+<!--
+Copyright 2022 Adobe
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+Adaptive Form File Input (v2)
+====
+Adaptive Form File input field component written in HTL.
+
+## Features
+
+* Drag and drop support
+* Custom icon for drag and drop
+* Provides the following type of input:
+  * file
+* Custom constraint messages for the above types
+* Styles
+
+### Use Object
+The Form File component uses the `com.adobe.cq.forms.core.components.models.form.FileInput` Sling Model for its Use-object.
+
+### Edit Dialog Properties
+The following properties are written to JCR for this Form File component and are expected to be available as `Resource` properties:
+
+1. `./jcr:title` - defines the label to use for this field
```

Comment:
fileInputDragDropText property is missing in README

Reply:
added